srcpkgs/pcsxr-pgxp/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'pcsxr-pgxp'
+pkgname=pcsxr-pgxp
+version=1.9.99
+revision=1
+lib32disabled=yes
+wrksrc="pcsxr-master"
+nostrip_files="pcsxr"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=/usr/lib
+ -DSND_BACKEND=sdl -DENABLE_CCDDA=ON -DUSE_LIBARCHIVE=ON -DUSE_LIBCDIO=ON -DOpenGL_GL_PREFERENCE=GLVND"
+hostmakedepends="pkg-config automake libtool intltool glib-devel gettext-devel nasm zip unzip"
+makedepends="libXv-devel libXtst-devel libcdio-devel SDL-devel gtk+3-devel libarchive-devel
+ libXv-devel gdk-pixbuf-devel SDL2-devel SDL-devel libXext-devel libpulseaudio libX11-devel libXxf86vm
+ zlib-devel libepoxy-devel glu-devel MesaLib-devel ffmpeg ffmpeg-devel"
```

Comment:
I think it's `libpulseaudio-devel` , not `libpulseaudio`

The same for `ffmpeg`

Reply:
there's pulseaudio-devel & libpulseaudio, no libpulseaudio-devel, at least not that I see...